src/helper/reader.rs

```diff
@@ -47,6 +48,7 @@ impl<'a> Reader<'a> {
         if (position + length) > self.size() {
             Err(::errors::NfsError::InvalidRangeSpecified)
         } else {
+            debug!("Reading data from file from {pos} to {len} ...", pos = position, len = length);
```

Comment:
`length` is not the ending position .. this gives a wrong impression. Reword as:
`Reading {len} bytes of data from file starting at offset of {pos} bytes ...`